### PR TITLE
`Jenkinsfile` cleanups

### PR DESCRIPTION
**Pull request by jglick**
https://github.com/jenkinsci/workflow-cps-plugin/pull/474/checks?check_run_id=4099320483 for example.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-buildPlugin(useAci: false, configurations: [
-  [ platform: "linux", jdk: "8", jenkins: null ],
-  [ platform: "windows", jdk: "8", jenkins: null ],
-  [ platform: "linux", jdk: "11", jenkins: "2.276", javaLevel: 8 ]
+buildPlugin(useContainerAgent: false, configurations: [
```
Comment (jglick) [inline Jenkinsfile:1]: Or
```suggestion
buildPlugin(useContainerAgent: true, configurations: [
```
if https://github.com/jenkinsci/workflow-cps-plugin/pull/355#issuecomment-624779205 has been cleared up.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
-buildPlugin(useAci: false, configurations: [
-  [ platform: "linux", jdk: "8", jenkins: null ],
```
Comment (jglick) [inline Jenkinsfile:2]: `jenkins: null` does nothing.

File: Jenkinsfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
-  [ platform: "windows", jdk: "8", jenkins: null ],
-  [ platform: "linux", jdk: "11", jenkins: "2.276", javaLevel: 8 ]
```
Comment (jglick) [inline Jenkinsfile:4]: https://github.com/jenkins-infra/pipeline-library/pull/205

File: Jenkinsfile (hunk 1 above)
Comment (car-roll) [reply Jenkinsfile:1]: Going to give it a shot...

**Review by car-roll — APPROVED** [review]

File: Jenkinsfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+buildPlugin(useContainerAgent: true, configurations: [
```
Comment (jglick) [inline Jenkinsfile:1]: Containerized Windows builds appear to be working for now anyway.

**Pull request merged**